.gitignore

```diff
@@ -9,6 +9,7 @@ erl_crash.dump
 /config/*.secret.exs
 .elixir_ls/
 /performance_manager/test.db
+/notebook
 
```

Comment:
clean up my git status

Reply:
totally fine.

in general i think additions here can just be pushed straight to origin main in their own small commit.